### PR TITLE
[KYUUBI #2156] Change log to reflect exactly why getting token failed

### LINKED ISSUE #2156: [Improvement] Change log to reflect exactly why getting token failed
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

In HiveDelegationTokenProvider.scala when forget to Specify Hive metastore configurations In $KYUUBI_HOME/conf/kyuubi-defaults.conf or Place  hive-site.xml into $KYUUBI_HOME/conf，the
metastoreUris will be empty，which will make the if condition return false，cause the client is null。

image

In HadoopCredentialsManager，provider.delegationTokensRequired will return false，the log 
 is Inaccurate，
image

### How should we improve?

“Check your configuration to see if security is disabled or not.”

should be changed to “Check your configuration to see if security is disabled or not.If security is enabled,some configuration of ${provider.serviceName} is missing”

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jiaoqingbo**
### _Why are the changes needed?_

fix #2156 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by jiaoqingbo** [comment]
cc @yaooqinn @pan3793 @zhouyifan279

**Comment by zhouyifan279** [comment]
> 

Thanks for the contribution! Would you like to write a doc about the required security configs of hdfs and hive. This will be very helpful if we can provide a link to this doc in the error log.

**Comment by jiaoqingbo** [comment]
> Thanks for the contribution! Would you like to write a doc about the required security configs of hdfs and hive. This will be very helpful if we can provide a link to this doc in the error log.

ok

File: kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopCredentialsManager.scala [hunk 1]
```diff
@@ -96,7 +96,8 @@ class HadoopCredentialsManager private (name: String) extends AbstractService(na
         val required = provider.delegationTokensRequired()
         if (!required) {
           warn(s"Service ${provider.serviceName} does not require a token." +
-            s" Check your configuration to see if security is disabled or not.")
+            s" Check your configuration to see if security is disabled or not." +
+            s" If security is enabled,some configuration of ${provider.serviceName} is missing")
```
Comment (yaooqinn) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopCredentialsManager.scala:100]:
```suggestion
            s" If security is enabled, some configurations of ${provider.serviceName} might be missing")
```

File: kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopCredentialsManager.scala [hunk 2]
```diff
@@ -96,7 +96,8 @@ class HadoopCredentialsManager private (name: String) extends AbstractService(na
         val required = provider.delegationTokensRequired()
         if (!required) {
           warn(s"Service ${provider.serviceName} does not require a token." +
-            s" Check your configuration to see if security is disabled or not.")
+            s" Check your configuration to see if security is disabled or not." +
```
Comment (yaooqinn) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopCredentialsManager.scala:99]:
```suggestion
            s" Check your configuration to see whether the security is disabled or not." +
```

**Review by yaooqinn — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master!

**Comment by zhouyifan279** [comment]
image

`Required Security Configs` layout is broken on my compute.

**Comment by pan3793** [comment]
@jiaoqingbo The doc format seems does not correct, would you please send a followup PR to fix it?
https://kyuubi.readthedocs.io/en/latest/security/hadoop_credentials_manager.html

BTW, you can follow https://kyuubi.readthedocs.io/en/latest/develop_tools/build_document.html to build the doc in your local environment.

**Comment by jiaoqingbo** [comment]
> image
> 
> `Required Security Configs` layout is broken on my compute.

sorry ，I will fix it

**Comment by jiaoqingbo** [comment]
> 

Ok

**Comment by jiaoqingbo** [comment]
> @jiaoqingbo The doc format seems does not correct, would you please send a followup PR to fix it? https://kyuubi.readthedocs.io/en/latest/security/hadoop_credentials_manager.html
> 
> BTW, you can follow https://kyuubi.readthedocs.io/en/latest/develop_tools/build_document.html to build the doc in your local environment.

Ok

**Comment by pan3793** [comment]
As this is a minor change to improve the user experience, I'm going to cherry-pick this one and #2165 to branch-1.5

**Pull request closed without merging**